projects/storefrontlib/src/cms-structure/routing/default-routing-config.ts

```diff
@@ -43,49 +43,12 @@ export const defaultStorefrontRoutesConfig: RoutesConfig = {
   },
 
   termsAndConditions: { paths: ['terms-and-conditions'] },
-  orders: {
-    paths: ['my-account/orders'],
-  },
-  orderDetails: {
-    paths: ['my-account/order/:orderCode'],
-    paramsMapping: { orderCode: 'code' },
-  },
-  orderGuest: {
-    paths: ['guest/order/:orderCode'],
-    paramsMapping: { orderCode: 'code' },
-  },
-  orderReturn: {
-    paths: ['my-account/order/return/:orderCode'],
-    paramsMapping: { orderCode: 'code' },
-  },
-  orderReturnConfirmation: {
-    paths: ['my-account/order/return/confirmation/:orderCode'],
-    paramsMapping: { orderCode: 'code' },
-  },
-  orderCancel: {
-    paths: ['my-account/order/cancel/:orderCode'],
-    paramsMapping: { orderCode: 'code' },
-  },
-  orderCancelConfirmation: {
-    paths: ['my-account/order/cancel/confirmation/:orderCode'],
-    paramsMapping: { orderCode: 'code' },
-  },
-  returnRequestDetails: {
-    paths: ['my-account/return-request/:returnCode'],
-    paramsMapping: { returnCode: 'rma' },
-  },
```

Comment:
Shouldn't we deprecate these instead of deletiing them?

Reply:
I am not sure for the default config. They are not in `public_api`.

Reply:
I suspect the current order feature will break if you remove these.

Reply:
I will revert it back